client/blocks/signup-form/p2.scss

```diff
@@ -1,133 +0,0 @@
-@import "calypso/assets/stylesheets/p2-extends";
-
-.is-p2-invite.invite-accept,
-.is-p2-signup .signup__step.is-user {
```

Comment:
There are a bunch of unused `.is-p2-signup` styles still in the codebase. In addition to removing them, we should be able to remove the `addP2SignupClassName` and `removeP2SignupClassName` helpers.

Reply:
Done in [32973a2](https://github.com/Automattic/wp-calypso/pull/101712/commits/32973a2591ee9bf01ffeb31e6a612882237c2056)